Content.Server/Mind/MindSystem.cs

```diff
@@ -61,6 +62,18 @@ public void InternalEjectMind(EntityUid uid, MindComponent? mind = null)
         mind.Mind = null;
     }
 
+    public void TransferMindProperties(MindComponent source, MindComponent target)
+    {
+        var props = typeof(MindComponent).GetProperties().ToList().Where(x => x.CanWrite);
+        string[] includedProps = { "ShowExamineInfo", "GhostOnShutdown", "PreventGhosting" };
+
+        foreach (var prop in props)
+        {
+            if (includedProps.Contains(prop.Name))
+                prop.SetValue(target, prop.GetValue(source));
+        }
+    }
```

Comment:
Mind code shouldn't be making its own implementation of component copying, especially not using reflection.